### DISCUSSION TITLE
VS Code - "lib" folder is missing when I'm creating a new project with  "forge init".

### DISCUSSION THREAD

**Discussion by RionNg** [Cyfrin/foundry-full-course-cu#1101] [Q&A] [answered]
Hello, everyone.

I'm encountering some issues with VS Code.

"lib" folder is missing when I'm creating a new project by entering  "_forge init_".
Some commands are not working, such as:
history -c

Do you have any suggestions on how to reinstall or troubleshoot VS Code on a Mac?  (Should I?)
Is reinstalling or resetting the entire VS Code a preferable approach?
What about the terminal?

I wish to redo/reinstall everything from 0, to ensure every step is doing correctly.
I have no idea what's the problem now, the VS Code or terminal.

Thank You!!

**Comment by 0xusmanf** [comment, marked as answer]
Read [this](https://github.com/Cyfrin/foundry-full-course-f23/discussions/201#discussioncomment-6324143) answer, it might be helpful.

And no I don't think you need to reinstall the VS code, this seems to be the problem with forge. Follow the instructions in the mentioned answer and hopefully, your issue will be resolved without reinstalling anything.

**Reply by RionNg** [reply]
It worked!!
I am new to this field. 
Thank you so much @usmanfarooq91

**Reply by 0xusmanf** [reply]
You are welcome, happy to help mate.
